locales/translation-de.json

```diff
@@ -285,15 +288,18 @@
       "name": "Die Adresse, unter der die Gruppe \"@softwerkskammer.org\" erreichbar ist.",
       "title": "Anzeigetext innerhalb der Site, ist auch Realname für E-Mails",
       "x_coord": "Waagerechte Position auf der Karte",
-      "y_coord": "Senkrechte Position auf der Karte"
+      "y_coord": "Senkrechte Position auf der Karte",
+      "contactTheOrganizers": "Auf der Gruppenseite wir ein Kontaktlink angezeigt, über den Besucher ohne Anmeldung Kontakt mit den Ansprechpartnern aufnehmen können."
     },
     "type": "Gruppenart",
     "x_coord": "X-Koord.",
     "y_coord": "Y-Koord."
   },
   "mailsender": {
+    "contact_persons_cannot_be_contacted": "Ansprechpartner können für diese Gruppe nicht kontaktiert werden.",
     "contents": "Inhalt",
     "copy_to_self": "Eine Kopie an mich senden",
+    "group_has_no_organizers": "Die Gruppe hat keine Ansprechpartner.",
```

Comment:
Gibt es das? Sollte es eigentlich nicht... Das sollte man vielleicht in Software verhindern.

Reply:
Wir haben es beim Testen hinbekommen.

Reply:
OK, ich habe mich hier zu schwammig ausgedrückt. Als ich diesen Text sah, dämmerte mir schon, dass das technisch möglich ist. Und ich sehe genau das als den Defekt an... Also sollte man in meinen Augen eher dort ansetzen und es nicht zulassen, dass eine Gruppe keinen Ansprechpartner hat. Dann könnt Ihr Euch auch die Sonderbehandlung sparen.

Reply:
Ich habe mal einen Issue dafür eingestellt.

Reply:
D.h. wir nehmen die Logik dafür ganz raus, da dies eher nicht auftritt bzw. bald gar nicht mehr passieren kann?

Reply:
Ihr könntet den Issue fixen, dann könnt Ihr hier die Logik rausnehmen ;-)

(andersrum wird das leider nix, weil zu hohes Explosionspotential.)

Reply:
Zum Thema "Hat keine Ansprechpartner": Bei einem manuellen Test sehe ich den Button auf der Gruppenseite, obwohl die Gruppe keinen Ansprechpartner hat. - Das zu unterdrücken wäre natürlich noch ein echtes Schmankerl!

Reply:
Habe ich umgestellt: Button wird nur noch angezeigt, wenn es auch Ansprechpartner gibt.